lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -104,5 +104,11 @@ public interface IFileSystem
         /// <returns></returns>
         /// <exception cref="IOException"></exception>
         IEnumerable<Uri> GetChildren(Uri directoryUri);
+
+        /// <summary>
+        /// Returns Url prefix 
+        /// </summary>
+        /// <returns></returns>
+        string UriPrefix { get; }
```

Comment:
This seems like a leaky abstraction. Why should user of IFileSystem abstraction have to care about UriPrefix. My expectation would be that the implementation transparently takes care of this. Is this just for testing?